mcodingbot/database/converters.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+from decimal import Decimal
+
+from apgorm import Converter
+
+
+class NumericC(Converter[Decimal, int]):
```

Comment:
What is C? This class needs a better name that makes it ambitious. It probably stands for Converter but C is used so much to refer to the language (like `c_int`) that's likely what people's first thought will be

Reply:
Should I use NumericInt, DecimalInt, NumericConverter, or DecimalConverter?

Reply:
Probably NumericConverter